### ISSUE TITLE
Error on adding applicationContext to addArgumentResolvers

### ISSUE THREAD

**Issue by napsterv** [tkaczmarzyk/specification-arg-resolver#92]
I'm trying to follow your github tutorial to get SpEL working.

```
import net.kaczmarzyk.spring.data.jpa.web.SpecificationArgumentResolver;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.support.AbstractApplicationContext;
import org.springframework.web.method.support.HandlerMethodArgumentResolver;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;

import java.util.List;

@Configuration
public class MvcConfig implements WebMvcConfigurer {
    private final AbstractApplicationContext applicationContext;

    @Autowired
    public MvcConfig(AbstractApplicationContext applicationContext) {
        this.applicationContext = applicationContext;
    }


    @Override
    public void addArgumentResolvers(List<HandlerMethodArgumentResolver> argumentResolvers) {
        argumentResolvers.add(new SpecificationArgumentResolver(applicationContext));
    }
}

```
But it gives me this error;

`error: incompatible types: AbstractApplicationContext cannot be converted to ConversionService`

I am using Java 11 and Spring Boot 2.3.2

Any help?

**Comment by jradlica** [comment]
@napsterv Are you sure the library version in your pom.xml is equal to 2.6.0 (or greater)? (SpEL support was introduced in 2.6.0)

**Comment by napsterv** [comment]
@jradlica I updated the library and the SpEL works perfectly. I was blindly following the code snippet on main repo page unaware that SpEL is in the latest update. I was on version 2.5.0. Thank you :)

A big thanks to @tkaczmarzyk and all the contributors. I come from Django background and this library works as a perfect substitute for django-filters in spring world. Thank you all :1st_place_medal: 

Closing :+1:
